src/Utilities/TypeTraits.hpp

```diff
@@ -212,6 +212,23 @@ struct negation : bool_constant<!B::value> {};
 template <typename... Ts>
 using void_t = void;
 
+// @{
```

Comment:
For example, adding these kinds of helpers should always be done in a separate commit :)